packages/scoped-elements/README.md

```diff
@@ -0,0 +1,276 @@
+# Scoped elements
+
+[//]: # 'AUTO INSERT HEADER PREPUBLISH'
+
+:::warning
+This is an experimental feature use it at your own risk and be sure to understand it's [limitations](#limitations).
+No bigger applications is using `scoped-elements` - so there is no prove yet if it works in production.
+This page focuses a lot on in depth explanations as it should help foster a discussion on scoping.
+:::
+
+Complex Web Component applications often are developed by several teams across organizations.
```

Comment:
as an aside, why does every line have a 'manual' line break? Why not just let the paragraph run?

Reply:
It's because my IDE setup. I could change it if it's a problem.